CHANGELOG.rst

```diff
@@ -1,3 +1,11 @@
+0.1.5 (2015-11-23)
```

Comment:
Prefer to review version bumps independent of the changes being made, because of decisions and discussion about minor vs major, etc. Also makes it clear a version bump is happening which avoids conflicts with other pulls that also think they deserve a version bump.